docs/architecture/decisions/0002-generating-unique-ids.md

```diff
@@ -0,0 +1,41 @@
+# 1. Record architecture decisions
+
+Date: 2023-03-16
+
+## Status
+
+Accepted
+
+## Context
+
+When users start an LPA either online or via the paper route, their data will be sent to and stored in this ID service data store. We need to create a unique ID to store alongside the users data, which case workers can then use to find LPA's by when actioning any future enquiries or objections.
+
+We must define the format and method of generating these IDs. This can change for private beta, but should preferably be something we don’t need to change until then.
+
+The IDs generated must:
```

Comment:
What needs are driving this? Are we talking unique ids or human readable reference numbers? These are potentially two different things.

Reply:
Yeah thats a good point, its more of a human readable reference number rather than an ID so i should make that clearer in the doc